### ISSUE TITLE
Documentation issue

### ISSUE THREAD

**Issue by shakisha** [docker-mailserver/docker-mailserver#3424]
### 📝 Preliminary Checks

- [X] I tried searching for an existing issue and followed the [debugging docs](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/) advice, but still need assistance.

### 👀 What Happened?

In the documentation there are instruction on how to setup the docker compose image, which are:

```
DMS_GITHUB_URL="https://github.com/docker-mailserver/docker-mailserver/blob/latest"
wget "${DMS_GITHUB_URL}/docker-compose.yml"
wget "${DMS_GITHUB_URL}/mailserver.env"
```

but doing this, the result will be:

```
--2023-07-13 08:44:42--  https://github.com/docker-mailserver/docker-mailserver/blob/latest/docker-compose.yml
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-07-13 08:44:42 ERROR 404: Not Found.

--2023-07-13 08:44:42--  https://github.com/docker-mailserver/docker-mailserver/blob/latest/mailserver.env
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-07-13 08:44:43 ERROR 404: Not Found.
```

### 👟 Reproduction Steps

just do the steps above

### 🐋 DMS Version

v12.1.0

### 💻 Operating System and Architecture

ubuntu

### ⚙️ Container configuration files

```yml
.
```

### 📜 Relevant log output

```Text
.
```

### Improvements to this form?

.

**Comment by casperklein** [comment]
Thanks for reporting, this issue has already been addressed.

Duplicate of: #3375 #3394 #3409 
Fixed in: #3337

To see the most current version of the documentation, [you have to select "edge" in the top left drop down menu](https://docker-mailserver.github.io/docker-mailserver/edge/usage/#get-all-files).

@georglauterbach @polarathene We are getting a lot issues of this kind. Shouldn't we make "edge" the default in our documentation, to prevent this and similar kind of issues in the future?

**Comment by polarathene** [comment]
@casperklein we switched to `:latest` for similar reasons.

This one is because old docs advise downloading files from the repo that no longer exist. We could copy/paste the section from edge into the latest 12.1 version of docs in the gh-pages branch, that would remedy this specific issue.

Alternatively making a new release but since development and release are managed by the same branch history we can't cherry pick. I assume a release may be a while off and relying on @georglauterbach availability (no pressure).

So easiest approach is probably manually patching the 12.1 docs, I could do that if you like?

**Comment by polarathene** [comment]
One other approach we've used in the past is to just pin an issue for awareness until we've made a new release, that has semi-worked in the past.

**Comment by georglauterbach** [comment]
Manually patching v12.1 is fine with me:)

v13.0.0 is blocked by the update to Debian 12, so a bit into the future.
